modules/help_editor/php/edit_help_content.class.inc

```diff
@@ -58,16 +58,22 @@ class Edit_Help_Content extends \NDB_Form
         if (isset($_REQUEST['helpID'])) {
             $helpID = htmlspecialchars($_REQUEST['helpID']);
         }
+        if (isset($_GET['parentID'])) {
+            $parentID = htmlspecialchars($_GET['parentID']);
+        }
         if (!empty($_REQUEST['section'])) {
             $helpID = HelpFile::hashToID(md5($_REQUEST['section']));
         }
         if (!empty($_REQUEST['section'])
             && $_REQUEST['subsection'] != 'undefined'
         ) {
-            $helpID = HelpFile::hashToID(md5($_REQUEST['subsection']));
+            $helpID   = HelpFile::hashToID(md5($_REQUEST['subsection']));
+            $parentID = HelpFile::hashToID(md5($_GET['section']));
         }
         $this->tpl_data['section']    = $safeSection;
         $this->tpl_data['subsection'] = $safeSubsection;
+        $this->tpl_data['helpID']     = $helpID ? $helpID : '';
+        $this->tpl_data['parentID']   = $parentID ? $parentID : '';
```

Comment:
Again, not a blocker, but these could use the elvis operator to make it a little cleaner (https://stackoverflow.com/questions/1993409/operator-the-elvis-operator-in-php)